tools/doc-generator/doclet/src/main/java/org/projectnessie/nessie/docgen/SmallryeConfigs.java

```diff
@@ -72,9 +72,12 @@ public List<SmallRyeConfigSection> buildConfigSections(DocletEnvironment env) {
       additionalPrefixes.forEach(
```

Comment:
There is imo a slight risk here that a prefix override (aka section name) could clash with an existing prefix. I didn't fix that.

Reply:
Wouldn't that mean that we have duplicate property keys?

Reply:
I fear something like this:

```java

@ConfigMapping(prefix = "foo")
public interface Clash {

  String bar();

  @ConfigItem(section = "foo.bar")
  String qix();

}
```